### PR TITLE
fix(clerk-js): Fix uneven profile card left/right spacing

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

An invisible scrollbar was adding more space to the right side of components like `<CreateOrganization/>` and `<UserProfile/>`. I've modified the `maxHeightScroller` to use `overflowY: auto` and `scrollbar-gutter: stable both-edges` to create the appropriate space in the opposite side.

### CODE REVIEW

File: packages/clerk-js/src/ui/styledSystem/common.ts [hunk 1]
```diff
@@ -194,12 +194,13 @@ const unstyledScrollbar = (t: InternalTheme) => ({
   '::-webkit-scrollbar-track': {
     background: 'transparent',
   },
+  scrollbarGutter: 'stable both-edges',
```
Comment (panteliselef) [inline packages/clerk-js/src/ui/styledSystem/common.ts:197]: I believe this is not available for Safari, will that be a problem ?

**Comment by nikosdouvlis** [comment]
Why is there a scroller in the first place? Can you share more context please?

File: packages/clerk-js/src/ui/styledSystem/common.ts (hunk 1 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/styledSystem/common.ts:197]: The alternative is `overflow: overlay` which is a legacy value.

**Comment by desiprisg** [comment]
@nikosdouvlis With `overflow: auto`, there will no longer be a scrollbar when the parent height is enough. I.e. it will appear in our modals and if the developer has a container that is limited in its height.
https://github.com/clerkinc/javascript/blob/abb076a171480722d3ee6c0ff116d3bda500ed3e/packages/clerk-js/src/ui/elements/ProfileCardContent.tsx#L41 This is the code that caused the scrollbar

**Review by anagstef — APPROVED** [review]

**Pull request merged**